spec/jobs/activity_tracking_job_spec.rb

```diff
@@ -59,22 +59,8 @@
       end
     end
 
-    it 'creates notifications when a comment is created' do
-      commentable = create(:issue)
-      create_list(:subscription, 2, subscribable: commentable)
-      trackable = create(:comment, commentable: commentable)
-      project = commentable.node.project
-
-      expect {
-        described_class.new.perform(
-          action: 'create',
-          project_id: project.id,
-          trackable_id: trackable.id,
-          trackable_type: trackable.class.to_s,
-          user_id: trackable.user.id
-        )
-      }.to change { Notification.count }.by(2)
-    end
+    include_examples 'creates notifications for comments in a', :issue
+    include_examples 'creates notifications for comments in a', :note
```

Comment:
as discussed in the call: this PR doesn't introduce any changes to the ActivityTracking code, so it's weird that we have to change its spec.

Another smell is that the responsibility for ensuring the behaviour of :note is put in the ActivityTracking spec, when it should be in some note-related spec. Otherwise this is only going to get uglier as we add more notifiable models.

Reply:
In fact I think we are already testing this in the right place, the `comment` model: https://github.com/dradis/dradis-ce/blob/master/spec/models/comment_spec.rb#L40